src/runner-configuration/machine-options.ts

```diff
@@ -1,8 +1,20 @@
+/**
+ * @see https://gitlab.com/gitlab-org/ci-cd/docker-machine/-/blob/main/commands/create.go
+ */
+export interface SharedCreateOptions {
+  /**
+   * Custom URL to use for engine installation
+   *
+   * @default https://releases.rancher.com/install-docker/20.10.21.sh
+   */
+  readonly engineInstallUrl?: string;
```

Comment:
Maybe sometime later we can change the URL to `https://get.docker.com`

Reply:
Yes, we can leave an issue open as reminder to remove the default override